src/Compilers/Core/Portable/PEWriter/CompilationOptionNames.cs

```diff
@@ -28,5 +29,6 @@ internal static class CompilationOptionNames
         public const string Nullable = "nullable";
         public const string Define = "define";
         public const string Strict = "strict";
+        public const string SourceFileCount = "source-file-count";
```

Comment:
The purpose of this property is to let us order the "real" documents at the beginning of the table, then allow documents added for `#line` directives to come at positions after this count so that we can skip them when rebuilding.